components/clp-package-utils/clp_package_utils/scripts/decompress.py

```
@@ -79,7 +81,7 @@ def handle_extract_file_cmd(
     # Validate extraction directory
     extraction_dir = pathlib.Path(parsed_args.extraction_dir).resolve()
     try:
-        validate_path_could_be_dir(extraction_dir)
+        validate_path_could_be_dir(extraction_dir, True)
```

Ruff (0.14.2)

Boolean positional value in function call

(FBT003)